int32 id
float32 distance
float32 theta
float32 yaw
geometry_msgs/Point32[] corners

